src/defaultContent/styles/main.css

```diff
@@ -326,6 +326,21 @@ h6 {
   color: var(--primary-color);
 }
 
+.app-header{
```

Comment:
do we need these styles in main css or can we move to application css?

Reply:
moved